src/test/java/org/greenbuttonalliance/gbaresourceserver/usage/repository/IntervalBlockRepositoryTest.java

```diff
@@ -98,31 +100,37 @@ public void findAll_returnsAll() {
 
 	@Test
 	public void entityMappings_areNotNull() {
-		IntervalBlock fullyMappedBlock = intervalBlockRepository.findById(UuidCreator.getNameBasedSha1(UuidCreator.NAMESPACE_URL, PRESENT_SELF_LINK)).orElse(null);
-		Assumptions.assumeTrue(fullyMappedBlock != null);
+		IntervalBlock fullyMappedIntervalBlock = intervalBlockRepository.findById(UuidCreator.getNameBasedSha1(UuidCreator.NAMESPACE_URL, PRESENT_SELF_LINK)).orElse(null);
+		Assumptions.assumeTrue(fullyMappedIntervalBlock != null);
 
-		// build functions to test mappings from IntervalBlock <--> IntervalReading <--> ReadingQuality
-		Function<IntervalBlock, Optional<Set<IntervalReading>>> blockToReadings = ib -> Optional.ofNullable(ib.getIntervalReadings());
-		Function<IntervalBlock, Optional<Set<ReadingQuality>>> blockToReadingQualities = blockToReadings.andThen(opt -> opt.flatMap(
+		/* Test bidirectional mappings all the way from IntervalBlock <--> IntervalReading <--> ReadingQuality here since IntervalReading and ReadingQuality don't have
+		their own repositories for which we're testing their individual mappings */
+		Function<IntervalBlock, Optional<Set<IntervalReading>>> blockToIntervalReadings = ib -> Optional.ofNullable(ib.getIntervalReadings());
+		Function<IntervalBlock, Optional<Set<ReadingQuality>>> blockToReadingQualities = blockToIntervalReadings.andThen(opt -> opt.flatMap(
 			readings -> readings.stream().findFirst().map(IntervalReading::getReadingQualities)
 		));
-		Function<IntervalBlock, Optional<IntervalReading>> blockToReadingReversed = blockToReadingQualities.andThen(opt -> opt.flatMap(
+		Function<IntervalBlock, Optional<IntervalReading>> blockToIntervalReadingReversed = blockToReadingQualities.andThen(opt -> opt.flatMap(
 			readingQualities -> readingQualities.stream().findFirst().map(ReadingQuality::getReading)
 		));
-		Function<IntervalBlock, Optional<IntervalBlock>> blockToBlockReversed = blockToReadingReversed.andThen(opt -> opt.map(
+		Function<IntervalBlock, Optional<IntervalBlock>> intervalBlockToIntervalBlockReversed = blockToIntervalReadingReversed.andThen(opt -> opt.map(
```

Comment:
Just wanted to make these variable names a little more specific

Reply:
Nice refactoring!!